src/io/g15.rs

```diff
@@ -227,7 +233,7 @@ impl G15Parser {
     /// Parse a g15_dumpplayer string via Regex search.
     /// We only extract useful data here, so drop most data.
     pub fn parse_g15(&self, g15_log: &str) -> Vec<G15Player> {
-        let mut players: Vec<G15Player> = vec![G15Player::new(); 34];
+        let mut players: Vec<G15Player> = vec![G15Player::new(); MAX_POSSIBLE_PLAYERS as usize];
```

Comment:
Should it be `MAX_POSSIBLE_PLAYERS` or `MAX_POSSIBLE_PLAUERS + 2`?

Reply:
yes it should be +2 :clown_face: